Error running git update for vvc

12:21:28   Running git update for vvc...
12:21:33 ┌ vvc git  ..................................... [Updates found]
12:21:33 ├ Running uninstall...
12:21:34 ├ Running cmake...
12:21:44 ├ Running build...
Likely error:
[152/191] Building CXX object source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncGOP.cpp.o
ninja: build stopped: subcommand failed.
[1/40] Building CXX object source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o
FAILED: source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o
C:\mingw\msys64\mingw64\bin\c++.exe  -DENABLE_SPLIT_PARALLELISM=1 -I../source/Lib/EncoderLib/. -I../source/Lib/CommonLib/. -I../source/Lib/CommonLib/.. -I../source/Lib/CommonLib/./x86 -I../source/Lib/CommonLib/../libmd5 -mthreads -mtune=generic -O2 -pipe -fopenmp -O3 -DNDEBUG   -Wall -fdiagnostics-show-option -Werror -Wno-sign-compare -Wno-class-memaccess -msse4.1 -mthreads -std=gnu++11 -Wno-array-bounds -MD -MT source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o -MF source\Lib\EncoderLib\CMakeFiles\EncoderLib.dir\EncCu.cpp.o.d -o source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o -c ../source/Lib/EncoderLib/EncCu.cpp
../source/Lib/EncoderLib/EncCu.cpp: In member function 'void EncCu::xCompressCUParallel(CodingStructure*&, CodingStructure*&, Partitioner&)':
../source/Lib/EncoderLib/EncCu.cpp:1242:35: error: 'PartitionerFactory' has not been declared
 1242 |     Partitioner* jobPartitioner = PartitionerFactory::get( *tempCS->slice );
      |                                   ^~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
build failed. Check C:/mingw/build/vvc-git/build-64bit/ab-suite.build.log
This isn't required for anything so we can move on.
12:23:22 ├ Running install...
Likely error:
      |                                   ^~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
[1/36] Building CXX object source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o
FAILED: source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o
C:\mingw\msys64\mingw64\bin\c++.exe  -DENABLE_SPLIT_PARALLELISM=1 -I../source/Lib/EncoderLib/. -I../source/Lib/CommonLib/. -I../source/Lib/CommonLib/.. -I../source/Lib/CommonLib/./x86 -I../source/Lib/CommonLib/../libmd5 -mthreads -mtune=generic -O2 -pipe -fopenmp -O3 -DNDEBUG   -Wall -fdiagnostics-show-option -Werror -Wno-sign-compare -Wno-class-memaccess -msse4.1 -mthreads -std=gnu++11 -Wno-array-bounds -MD -MT source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o -MF source\Lib\EncoderLib\CMakeFiles\EncoderLib.dir\EncCu.cpp.o.d -o source/Lib/EncoderLib/CMakeFiles/EncoderLib.dir/EncCu.cpp.o -c ../source/Lib/EncoderLib/EncCu.cpp
../source/Lib/EncoderLib/EncCu.cpp: In member function 'void EncCu::xCompressCUParallel(CodingStructure*&, CodingStructure*&, Partitioner&)':
../source/Lib/EncoderLib/EncCu.cpp:1242:35: error: 'PartitionerFactory' has not been declared
 1242 |     Partitioner* jobPartitioner = PartitionerFactory::get( *tempCS->slice );
      |                                   ^~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
install failed. Check C:/mingw/build/vvc-git/build-64bit/ab-suite.install.log
This isn't required for anything so we can move on.
12:23:31 ├ /local64/bin-video/vvc/EncoderApp.exe  . [Not found]
12:23:31 └ vvc-git  ............................................ [Failed]

Try deleting '/build/vvc-git' and start the script again.
If you're sure there are no dependencies <Enter> to continue building.
Close this window if you wish to stop building.

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -1655,7 +1655,7 @@ if [[ $vvc = y ]] &&
     _notrequired="true"
     # install to own dir because the binaries' names are too generic
     do_cmakeinstall -DCMAKE_INSTALL_BINDIR="$LOCALDESTDIR"/bin-video/vvc \
-        -DBUILD_STATIC=on -DSET_ENABLE_SPLIT_PARALLELISM=ON -DENABLE_SPLIT_PARALLELISM=ON
+        -DBUILD_STATIC=on -DSET_ENABLE_SPLIT_PARALLELISM=ON -DENABLE_SPLIT_PARALLELISM=OFF
     do_checkIfExist
     unset _notrequired
 fi

